src/loadNamespaces.tsx

```diff
@@ -43,12 +44,22 @@ export default async function loadNamespaces(
   const defaultLoader: LocaleLoader = () => Promise.resolve({})
   const pageNamespaces =
     (await Promise.all(
-      namespaces.map((ns) =>
-        typeof conf.loadLocaleFrom === 'function'
-          ? conf.loadLocaleFrom(__lang, ns)
-          : defaultLoader(__lang, ns)
+      namespaces.map(
+        (ns) =>
+          new Promise<I18nDictionary>((resolve) => {
+            if (typeof conf.loadLocaleFrom === 'function') {
+              conf
+                .loadLocaleFrom(__lang, ns)
+                .then(resolve)
+                .catch(() => resolve({}))
+            } else {
+              defaultLoader(__lang, ns)
+                .then(resolve)
+                .catch(() => resolve({}))
```

Comment:
The `defaultLoader` is defined on line 44 and is just a `Promise.resolve`, in this case the catch is not necessary and we reduce the library bytes if we remove it 😁